components/camel-leveldb/pom.xml

```diff
@@ -37,7 +37,9 @@
             org.apache.camel*;version=${camel-version}
         </camel.osgi.export>
         <camel.osgi.import>
-            *
+            *,
+            org.iq80.leveldb.impl;resolution:=optional,
+            org.fusesource.leveldbjni;resolution:=optional
```

Comment:
Are you sure that it does what you expect? Have you checked the generated manifest because IIRC, the star should always be the last

Reply:
I changed it although generated Manifest is identical